### DISCUSSION TITLE
Enable\add donations to the repo?

### DISCUSSION THREAD

**Discussion by redtrillix** [rejetto/hfs#289] [General]
I've been using your past and present HFS projects, and I have the financial ability to donate now.
Is there a place I can do this or is it possible for this to be set up?

**Comment by rejetto** [comment]
on the old hfs website you'll find this same link
https://www.paypal.com/donate/?hosted_button_id=HC8MB4GRVU5T2

I will have to re-do the website when HFS 3 becomes "official"

**Reply by Kuca-cara** [reply]
Bro just put up the donation link now. 
I just spent half an hour looking for it, then searching this forum for a "donation" thread, then signing up to GitHub... just to leave this comment.

I'm in the exact same position as @redtrillix 

You'll see my donation come thru in a couple mins.

Thank you for developing this. Wow. What an upgrade!

**Reply by rejetto** [reply]
thank you very much for your donation @Kuca-cara !
At the moment the "donation" link is only visible on the main page of this repository, when you click https://github.com/rejetto/hfs and it's called "sponsor" (not customizable).
I can add another link on the website https://rejetto.com/hfs/
Maybe I can also add a small icon heart on the admin-panel.
Where would you expect to see it? What's the first place where you would look for it?
